.eslintrc.yml

```diff
@@ -101,6 +101,9 @@ rules:
     - error
     - never
     - onlyEquality: true
+  prefer-const:
+    - error
+    - destructuring: all
```

Comment:
Changed to `destructuring: all` so eslint doesn't say `z` should be const in `let [x, y, z] = foobar();` if `x` and `y` need to be non-const. I'd rather have `let [x, y, z] = foobar();` syntax rather than some hack to make prefer-const happy in this case.